approved_tags.yaml

```diff
@@ -0,0 +1,103 @@
+---
+- "accessibilità"
+- "ia"
+- "android"
+- "api"
+- "apl"
+- "arte"
+- "domanda"
+- "assembly"
+- "audio"
+- "libro"
+- "browsers"
+- "c"
+- "c++"
+- "clojure"
+- "scienzecognitive"
+- "compilatori"
+- "informaticateorica"
+- "crittografia"
+- "criptovalute"
+- "css"
+- "cultura"
```

Comment:
questa l'ho sempre vista un po' critica. La dividerei in più tag: politica, società, etica, cultura IT e volendo altre.

Reply:
Per ora la divido in `"politica"`, `"società"`, `"etica"` e `"culturait"`, ma chiedo sul gruppo se qualcuno propone altro da aggiungere.

Reply:
Come detto sul gruppo, io sostengo che o si adopera il generico `cultura`, o si adoperano etichette specifiche come quelle suggerite da @chobeat .